docs/13-walkthroughs/5-big-bang.md

```diff
@@ -0,0 +1,182 @@
+set -o history
+```
+
+Now navigate to the `examples/big-bang` folder within the Zarf repository you cloned in the [pre-requisites](#pre-requisites) section.
+
+### Configure Big Bang
+
+Within the `examples/big-bang` folder you will see a `zarf.yaml` that has the following [component](../4-user-guide/2-zarf-packages/2-zarf-components.md) defined:
+
+```yaml
+components:
+  - name: bigbang
+    required: true
+    extensions:
+      bigbang:
+        version: 1.52.0
+        skipFlux: false
+        valuesFiles:
+          - config/minimal.yaml #turns on just istio
+          - config/ingress.yaml # adds istio certs for *.bigbang.dev
+          - config/kyverno.yaml # turns on kyverno
+          - config/loki.yaml # turns on loki and monitoring
+          - config/patch-images.yaml # fixes upstream bug in BB
+```
+
+This component uses the `bigbang` extension to define the version of Big Bang to use and the values files to apply.  Feel free to inspect and configure the values.yaml files as you wish and to learn more about Big Bang's configuration see their values guide: https://docs-bigbang.dso.mil/1.53.0/docs/guides/using-bigbang/values-guide/
+
+:::note
+
+The `valuesFiles` are applied from top to bottom and will apply the last value that was provided for any given key.
+
+:::
+
+:::note
+
+Versions of Big Bang prior to `1.54.0` require the `config/patch-images.yaml` values file to be included.
+
+:::
+
+
+### Package Big Bang
+
+When you're ready to continue you can create a Big Bang package by running the following command in `examples/big-bang`:
+
+```bash
+zarf package create
+```
+
+Now wait for the package creation to complete and you should see a `zarf-package-big-bang-example-amd64-x.x.x.tar.zst` file in the directory.
+
+
+## Package Deployment
+
+The below section covers deploying the Big Bang package from the previous section:
+
+### Initialize Zarf
+
+Before you can deploy the Big Bang package you must first initialize Zarf on the cluster you created in the [pre-requisites](#pre-requisites) section.  To do so you can run the following:
+
+```bash
+# Initialize Zarf (interactively)
+zarf init
+# Make these choices at the prompts
+# ? Do you want to download this init package? Yes
+# ? Deploy this Zarf package? Yes
+# ? Deploy the k3s component? No
+# ? Deploy the logging component? No
+# ? Deploy the git-server component? Yes
+
+# (Optional) Inspect the results
+zarf tools k9s
+```
+
+:::note
+
+The `git-server` component is required by Big Bang as it uses it as a source for Flux deployments.
+
+:::
+
+
+### Deploy Big Bang
+
+Now you are ready to deploy Big Bang, and can do so with the following in the `examples/big-bang` directory:
+
+```bash
+# Deploy Big Bang (interactively)
+./zarf package deploy
+# Make these choices at the prompts
+# ? Choose or type the package file [tab for suggestions] zarf-package-big-bang-example-amd64-x.x.x.tar.zst
+# ? Deploy this Zarf package? Yes
+```
+
+### See the results
```

Comment:
```suggestion
### See The Results
```

All other headers are fully capitalized, this is something that would be good to define in our docs style guide @Madeline-UX

Reply:
Agreed - Will add all headers to be in title caps to docs style guide